### ISSUE TITLE
Auto-start Docker

### ISSUE THREAD

**Issue by shanejansen** [shanejansen/touchstone#31]
Touchstone will not start if Docker isn't running. It would be helpful if Touchstone automatically started Docker for you

**Comment by ScottFreeCode** [comment]
Does this depend on which docker backend any given system uses? Which may not even depend on the project so much on whether it's being run on a developer's machine or a build/CI server and which OS (and/or OS distro) that is?

**Comment by shanejansen** [comment]
@ScottFreeCode Good point. I was originally thinking this would just be for develop mode, but maybe this could be expanded to included run mode if some build service don't leave docker running
